dnf-docker-test/features/steps/test_behave.py

```diff
@@ -64,6 +65,14 @@ def diff_package_lists(a, b):
         map(_right_decorator, list(sb - sa)))
 
 
+def diff_query_lists(sack_a, sack_b):
+    """ Computes diff between sack_a and sack_b """
+    sa, sb = list(sack_a.query().installed()), list(sack_b.query().installed())
+    rem = [x for x in sa if x not in sb]
+    inst = [x for x in sb if x not in sa]
```

Comment:
you mean set of packages (`set(sack_a.query().installed())`) or really set of sacks?